src/clients/python/library/grpcclient.py

```diff
@@ -893,6 +894,11 @@ def infer(self,
             model-specific action such as terminating the request. If not
             provided, the server will handle the request using default setting
             for the model.
+        client_timeout : int
+            The maximum end-to-end time, in seconds, the request is allowed
```

Comment:
Is this really seconds? That seems very coarse.

Reply:
> Parameters
> request – The request value for the RPC.
> timeout – An optional duration of time in seconds to allow for the RPC.
> metadata – Optional metadata to be transmitted to the service-side of the RPC.
Link: [here](https://grpc.github.io/grpc/python/_modules/grpc.html#UnaryUnaryMultiCallable.__call__)
Looks like it.. The library also doesn't accept the fractional value for timeout. So, yes the minimum timeout that we can provide is 1s.